chapters/fr/chapter1.md

```diff
@@ -0,0 +1,325 @@
+---
+title: 'Chapitre 1 : Recherche de mots, phrases, noms et concepts'
+description:
+  'Ce chapitre te présente les bases du traitement de texte avec spaCy. Tu vas
+  découvrir les structures de données, comment utiliser les modèles
+  statistiques, et comment les employer pour prédire des caractéristiques
+  linguistiques dans ton texte.'
+prev: null
+next: /chapter2
+type: chapter
+id: 1
+---
+
+<exercise id="1" title="Présentation de spaCy" type="slides">
+
+<slides source="chapter1_01_introduction-to-spacy" start="0:165" end="3:01">
+</slides>
+
+</exercise>
+
+<exercise id="2" title="Prise en main">
+
+Commençons à utiliser spaCy ! Dans cet exercice, tu vas pouvoir essayer quelques
+uns des 55+ [langages disponibles](https://spacy.io/usage/models#languages).
+
+### Partie 1 : Anglais
+
+- Importe la classe `English` depuis `spacy.lang.en` et crée l'objet `nlp`.
+- Crée un `doc` et affiche son texte.
+
+<codeblock id="01_02_01"></codeblock>
+
+### Partie 2 : Allemand
+
+- Importe la classe `German` depuis `spacy.lang.de` et crée l'objet `nlp`.
+- Crée un `doc` et affiche son texte.
+
+<codeblock id="01_02_02"></codeblock>
+
+### Partie 3 : Espagnol
```

Comment:
I think we should replace one of these with a French example, because it'd be weird not to. I'd also swap it so that French is first (I did the same for German).

You get to decide which language to replace and whether French users should do German or Spanish 😛

Reply:
Done with French example followed by English and Spanish example (most spoken foreign languages by French people)